src/main/java/ru/aston/mapper/PostMapper.java

```diff
@@ -0,0 +1,27 @@
+package ru.aston.mapper;
+
+import ru.aston.dto.PostDto;
+import ru.aston.entity.Post;
+
+public class PostMapper {
+
+    private PostMapper() {
+    }
+
+    public static Post toPost(PostDto postDto) {
```

Comment:
маппер это не утилитный класс, не нужно делать static методы
```suggestion
    public Post fromDto(PostDto postDto) {
```